packages/graphql-client/documents/delegates.graphql

```diff
@@ -1,15 +1,28 @@
-  query GetDelegates($after: String, $before: String, $orderBy: String = "votingPower", $orderDirection: String = "desc") {
-    accountPowers(orderBy: $orderBy, orderDirection: $orderDirection, limit: 10, after: $after, before: $before, where: {votingPower_gt: 0}) {
-      items {
-        votingPower
-        accountId
-        delegationsCount
-      }
-      pageInfo {
-        endCursor
-        hasNextPage
-        hasPreviousPage
-        startCursor
-      }
+query GetDelegates(
```

Comment:
The GraphQL query now includes accountId_in filter but doesn't handle the case where addresses is undefined/null. When addresses is undefined, the GraphQL filter accountId_in: $addresses will likely cause the query to return no results instead of returning all results. This breaks existing functionality when no address filter is applied.